ga4gh/datamodel/reads.py

```diff
@@ -75,3 +89,59 @@ def toProtocolElement(self):
         readGroup.referenceSetId = None
         readGroup.sampleId = None
         return readGroup
+
+    def getReadAlignments(self, referenceName, referenceId, start, end):
```

Comment:
Maybe we should give these parameters defaults of `None` to make the testing code a little easier?